template/README.md.jinja

```diff
@@ -1,3 +1,18 @@
 # {{ short_project_name }}
 
 Generated via copier from github.com/catalystsquad/copier-go-cobra-app
+
+In order to run this code you will need the following on your path:
+- buf
+- Go 1.21+
+- Node 18+ (and npm of course)
+- Java of some kind
```

Comment:
java?

Reply:
For the openapi-client-generator, I'll add that not